client/view/QueryLink/queryLink.js

```diff
@@ -10,6 +12,10 @@ links.forEach(item => {
     } else {
       setFormValues({ query: item.innerText.trim() })
     }
+
     submitForm()
+
+    queryContainer.scrollTo({ top: 0 })
+    queryContainer.scrollIntoView()
```

Comment:
Why do we have 2 ways of scrolling?

Reply:
We have 2 scrollable areas

1. Scroll in `queryContainer` (with rules `min-height 100vh; overflow: auto`). On phone screen our scroll position is at the bottom when we are under `queryContainer`.

2. Scroll to top of `queryConainer` in viewport

Did you get it? :)